modules/end-to-end/pages/building-tekton-tasks.adoc

```diff
@@ -0,0 +1,298 @@
+= Building Tekton tasks as bundles in {ProductName}
+
+This document provides a step-by-step guide on how to onboard your Tekton tasks to {ProductName}, enabling their integration into your build pipelines.
+
+
+== Prerequisites
+
+Before you begin, ensure you have the following:
+
+* A Github repository containing your Tekton tasks with a catalog structure, containing all the additional files (README, TROUBLESHOOTING, etc…)
+
+----
+tasks/
+└── echo/
+    ├── 0.1/
+    │   ├── echo.yaml
+    │   ├── README.md
+    │   ├── USAGE.MD
+    │   └── TROUBLESHOOT.md
+    └── 0.2/
+        ├── echo.yaml
+        └── README.md
+----
+
+* {ProductName} Github application installed in the repository xref:installing:github-app.adoc[GitHub App]
+
+* A quay organization to be used with on-pr and push pipelines.
+
+* Public repositories within the quay organization to push the bundles to when using the on-pr and push pipelines.
+
+* A quay organization for the released bundles.
+
+* A public repository within the release organization for the trusted tasks list.
```

Comment:
Is this something that users would need to configure or is it just a property of the release pipelines? i.e. any SRE, etc. would have to provide this ability for you to release to?

Reply:
Users will need to configure it.
The release process will release the bundle to a repository but it will also update the trusted tasks list and a repository for the trusted tasks list should be created before.